components/backup-stream/src/subscription_manager.rs

```diff
@@ -543,6 +543,7 @@ where
         let tx = tx.unwrap();
         // tikv_util::Instant cannot be converted to std::time::Instant :(
         let start = std::time::Instant::now();
+        info!("Scheduing subscription."; utils::slog_region(&region), "after" => ?backoff, "handle" => ?handle);
```

Comment:
Maybe too verbose? I find there lots of logs from PiTR in tikv.log.

Reply:
Generally this should be print once retrying. But I guess with the async-backtrace we can make it debug level.